### PR TITLE
fix(FX-4572): contact gallery button artwork page

### PR DESCRIPTION
**Pull request by gkartalis**
The type of this PR is: **Fix**

This PR solves [FX-4572]

### Description

- [x] contact gallery button is displayed when partner has field `isInquireable: true`
- [x] avoids `Contact Gallery` button duplication by checking if artwork isInquireable (which is when we display the big cta)

[review app](https://artwork-partner-contact.artsy.net/)

#### Screenshots

|Before|After|
|---|---|
|Screenshot 2023-02-13 at 13 24 36|Screenshot 2023-02-13 at 13 24 24|

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FX-4572]: https://artsyproduct.atlassian.net/browse/FX-4572?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarPartnerInfo.jest.tsx [hunk 1]
```diff
@@ -122,78 +124,9 @@ describe("ArtworkSidebarPartnerInfo", () => {
...
-      mockUseSystemContext.mockImplementation(() => ({
-        featureFlags: { "conversational-buy-now": { flagEnabled: false } },
-      }))
-
-      renderWithRelay({
-        Artwork: () => ({
-          ...ArtworkFromPartnerWithLocations,
-          isOfferable: true,
-          isAcquireable: true,
+          partner: {
```
Comment (dimatretyak) [inline src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarPartnerInfo.jest.tsx:127]: **suggestion**: update the description for test cases ([here](https://github.com/artsy/force/blob/352f08365536e6147e9cbcab48ac7cf0087084a7/src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarPartnerInfo.jest.tsx#L110) and [here](https://github.com/artsy/force/blob/352f08365536e6147e9cbcab48ac7cf0087084a7/src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarPartnerInfo.jest.tsx#L110))

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx [hunk 2]
```diff
@@ -87,7 +73,7 @@ const ArtworkSidebarPartnerInfo: React.FC<ArtworkSidebarPartnerInfoProps> = ({
           )}
         </PartnerContainer>
 
-        {shouldRenderContactGalleryCTA && (
+        {!!partner.isInquireable && (
```
Comment (dimatretyak) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx:76]: **question**: should we hide `Contact Gallery` button in [ArtworkSidebarCommercialButtons](https://github.com/artsy/force/blob/main/src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx#L382-L395) in this case?

**Review by dimatretyak — APPROVED** [review]

File: src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarPartnerInfo.jest.tsx (hunk 1 above)
Comment (gkartalis) [reply src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarPartnerInfo.jest.tsx:127]: 🙈

Comment (gkartalis) [reply src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarPartnerInfo.jest.tsx:127]: done, good catch 🙈

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx (hunk 2 above)
Comment (gkartalis) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx:76]: Good question, first thing that comes to mind is that we don't need to do it, but looking into it now you might be right

Comment (gkartalis) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx:76]: You are right, found one example of a sold artwork that with the new implementation has the contact gallery  button plus this button, so two contact gallery buttons 🤔 will bring it up with design

Comment (gkartalis) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx:76]: [artwork link](https://www.artsy.net/artwork/casey-van-loon-nicos-blanket) for reference, will spawn a review app so we can share etc

Comment (gkartalis) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx]: opened a thread and resolved it [here](https://artsy.slack.com/archives/C9SATFLUU/p1676296699409339), pushed the new changes

Comment (gkartalis) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx]: great catch @dimatretyak

**Review by dimatretyak — APPROVED** [review]

**Pull request merged**